guest-profile-metadata/sharingRules/Experience__c.sharingRules

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should the sharing rules be in the metadata? Asking b/c they are listed in `.forceignore`, so unclear about the reason for having them.

Reply:
These 4 types in `.forceignore` because they are tied to the profile and we can't retrieve them in the source format because of the profile:
- profiles
- profilePasswordPolicies
- profileSessionSettings
- sharingRules

This mirrors what we did with E-Bikes.